src/commands/test.ts

```diff
@@ -42,6 +42,12 @@ export async function testSolution(uri?: vscode.Uri): Promise<void> {
                 detail: "Test with the written cases in file",
                 value: ":file",
             },
+            {
+                label: "$(refresh) Re- run last...",
```

Comment:
UI: Use this icon please: `$(debug-restart)`

reference: https://microsoft.github.io/vscode-codicons/dist/codicon.html